### ISSUE TITLE
Core dump when most recent project does not exist

### ISSUE THREAD

**Issue by cranndarach** [olivierkes/manuskript#443]
Hello, I ran into a problem when opening Manuskript. I have moved the most recently opened project to a new folder, and since the file in the old location no longer exists, Manuskipt core dumps on startup.

Here's the output from calling it from the terminal:
```sh
$ manuskript

Debug: Web rendering engine used: QWebEngineView
Running manuskript version 0.7.0.
Note: No translator found or loaded for locale en_US.
The file /home/[...]/proposal.msk does not exist. Try again.
Traceback (most recent call last):
  File "/opt/manuskript/bin/../manuskript/ui/welcome.py", line 74, in showEvent
    self.mw.loadProject(last)
  File "/opt/manuskript/bin/../manuskript/mainWindow.py", line 557, in loadProject
    self.tr("The file {} does not exist. Try again.", importance=3).format(project))
TypeError: tr(self, str, disambiguation: str = None, n: int = -1): 'importance' is not a valid keyword argument
Fatal Python error: Aborted

Current thread 0x00007fb7528b2740 (most recent call first):
  File "/opt/manuskript/bin/../manuskript/main.py", line 87 in launch
  File "/opt/manuskript/bin/../manuskript/main.py", line 102 in run
  File "/usr/bin/manuskript", line 13 in <module>
[1]    26751 abort (core dumped)  manuskript
```

(I know that I can work around this by temporarily moving the project back to its old home, but I figured it's worth reporting anyway.)

**Comment by cranndarach** [comment]
Update: It looks like it was an issue with the `importance` argument being passed to `tr()`, and has been fixed in 0.8.0.

**Comment by gedakc** [comment]
Thank you for closing the report upon discovering it was fixed in 0.8.0 with #389.
